File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/PriorityModel.ts
import { SalesOrderModel } from "./salesOrderModel";

export interface PriorityModel {
    priorityId: number;
    priorityNameAr: string;
    priorityNameEn: string;
    icon: string;
    color: string;
    displayOrder: number | null;
    isActive: boolean | null;
    canEdit: boolean | null;
    canDelete: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;

}

File: SofiForce.Models/Models/EntityModels/PriorityModel.cs
using System;

namespace Models
{
    public class PriorityModel
    {
        public int PriorityId { get; set; }
        public string PriorityCode { get; set; }
        public string PriorityNameAr { get; set; }
        public string PriorityNameEn { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? IsActive { get; set; }
        public bool? CanEdit { get; set; }
        public bool? CanDelete { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}
